README.md

```diff
@@ -1,23 +1,37 @@
-= errtrace
-:toc: preamble
-:idprefix:
-:idseparator: -
-:tabsize: 4
+# errtrace
 
-image::assets/logo.png[errtrace logo,300,align="center"]
+<div align="center">
 
-// FYI: ":" makes the images inline. "::" above makes it a block.
-image:https://github.com/bracesdev/errtrace/actions/workflows/ci.yml/badge.svg["CI status", link="https://github.com/bracesdev/errtrace/actions/workflows/ci.yml"]
-image:https://pkg.go.dev/badge/braces.dev/errtrace.svg["Go Reference", link="https://pkg.go.dev/braces.dev/errtrace"]
-image:https://codecov.io/gh/bracesdev/errtrace/graph/badge.svg?token=KDY04XEEJ9["Code Coverage", link="https://codecov.io/gh/bracesdev/errtrace"]
+![errtrace logo](assets/logo.png)
 
-== Introduction
+</div>
+
+!["CI status"](https://github.com/bracesdev/errtrace/actions/workflows/ci.yml/badge.svg)
+!["Go Reference"](https://pkg.go.dev/badge/braces.dev/errtrace.svg)
+!["Code Coverage"](https://codecov.io/gh/bracesdev/errtrace/graph/badge.svg?token=KDY04XEEJ9)
+
+- [Introduction](#introduction)
+  - [Features](#features)
+  - [Try it out](#try-it-out)
+  - [Why is this useful](#why-is-this-useful)
+- [Installation](#installation)
+- [Usage](#usage)
+  - [Manual instrumentation](#manual-instrumentation)
+  - [Automatic instrumentation](#automatic-instrumentation)
+- [Performance](#performance)
+- [Caveats](#caveats)
+  - [Error wrapping](#error-wrapping)
+  - [Safety](#safety)
+- [Contributing](#contributing)
+- [Acknowledgements](#acknowledgements)
+- [License](#license)
+
+## Introduction
 
-[WARNING]
 errtrace is extremely experimental.
 Use it at your own risk.
 
-errtrace is an experimental package to trace an error's return path --
+errtrace is an experimental package to trace an error’s return path --
 the return trace -- through a Go program.
```

Comment:
ends up repeating the experimental, should we just drop the warning sentence, and make the "experimental" bold in this sentence?